code/src/components/Buttons.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
```

Comment:
A tip to be able to see the class name in the console:  "styled-components/macro (but you probably knew it)